examples/vic-gov-au/test/e2e/integration/core-modules/landingPage/AnchorLink.feature

```diff
@@ -1,14 +1,15 @@
 Feature: Anchor link
 
-  When user request a url with anchor link,
-  who should see the page scroll to the target.
-
-  Scenario: Visit a url with anchor link
-    Given I visit the page "/demo-landing-page#accordion-header-basic"
-    And the anchor linked item goes to the top of the page
+  As a citizen I need to be able to scroll to a section in the page content when clicking an anchor link
 
   Scenario: Visit a url with anchor link via link
     Given the "/anchorlink-test" page exists with fixture "page/anchorlinktest" data
     When I visit the page "/anchorlink-test"
     And I click the link "/demo-landing-page#accordion-header-basic"
-    Then the anchor linked item goes to the top of the page
+    And I wait for 10 seconds
```

Comment:
Need to put a wait here as the step definition `Then the page should scroll to` wasn't waiting for page load for some reason.